staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -43,5 +43,5 @@ func init() {
 // To add a new feature, define a key for it above and add it here. The features will be
 // available throughout Kubernetes binaries.
 var defaultKubernetesFeatureGates = map[utilfeature.Feature]utilfeature.FeatureSpec{
-	StreamingProxyRedirects: {Default: true, PreRelease: utilfeature.Beta},
+	StreamingProxyRedirects: {Default: true, PreRelease: utilfeature.GA},
```

Comment:
Fair point. The only precedent I see for features graduating to GA is `ExternalTrafficLocalOnly` which flipped to GA, but left the flag checks in. I can do that here, but I assume we're going to want to clean these up eventually. Maybe specifying a flag for a GA feature should trigger a warning message that states that the feature gate will be removed in a future release?

/cc @jlowdermilk